docker-compose.yml

```diff
@@ -0,0 +1,42 @@
+version: '3'
+
+services:
+  keycloak:
+    container_name: callisto-ui-keycloak-localdev
+    healthcheck:
+      test: ['CMD', 'curl', '-f', 'http://localhost:8080/auth']
+      start_period: 30s
+    image: jboss/keycloak:15.0.2
```

Comment:
This might be in the pipeline of work for improving the Dev experience so feel free to ignore / close this comment... this version of Keycloak isn't compatible with some Macs (M1) and we have to rebuild with a local compatible version e.g. image: wizzn/keycloak:14